test/TestAssets/TestProjects/TestProjectWithConditionOnGlobalProperty/OtherTestProject/OtherTestProject.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), testAsset.props))\testAsset.props" />
+
+	<PropertyGroup>
```

Comment:
Nit: Is it normal to have tabs?

Reply:
I copied from existing test assets, e.g:

https://github.com/dotnet/sdk/blob/main/test/TestAssets/TestProjects/TestProjectWithMultipleTFMsSolution/OtherTestProject/OtherTestProject.csproj

Reply:
We should consider ensuring formatting is aligned with .editorconfig which seems to be using two spaces for csproj. cc @mariam-abdulla